packages/cxl-ui/src/components/cxl-notification.js

```diff
@@ -5,6 +5,7 @@ export class CXLNotification extends Notification {
     return 'cxl-notification';
   }
 
+  // Provide a default renderer which appends all child nodes to the notification root element.
```

Comment:
Well I can see that from the code, but what's the need?

Reply:
> Well I can see that from the code, but what's the need?

Maybe this clears it: https://lit.dev/docs/components/shadow-dom/#slots

Reply:
Updated to reflect the need for this strategy as well as a reference to the Slack discussion (https://cxlworld.slack.com/archives/C01JABH8AHX/p1662121897793169).

Reply:
@anoblet maybe add this too: https://lit.dev/docs/components/shadow-dom/#slots

Reply:
@saas786 I can add it you would like. Though we aren't using slots here. There's not `slot` node or `slot name=""` node. All we are doing is moving direct children to another node.

Reply:
> @saas786 I can add it you would like. Though we aren't using slots here. There's no `slot` or `slot name=""` node. All we are doing is moving direct children to `vaadin-notification-card`.

Not specifically because of `slots`, but rather DOM rendering point of view, as far as I am aware, child elements won't render otherwise.

Reply:
Updated